src/exporters/srtm.py

```diff
@@ -70,7 +70,8 @@ def export(self, region_name: str = 'kenya',
                 elevation.clip(bounds=self._region_to_tuple(region),  # type: ignore
                                output=output_tif.resolve().as_posix(),
                                product=product,
-                               max_download_tiles=max_download_tiles)
+                               max_download_tiles=max_download_tiles,
+                               margin='1')
```

Comment:
what does `mrgin` do?

Reply:
so we define the region to download. The margin downloads an extra `margin` of lat / lon to download.